### ISSUE TITLE
Chat :  Unblocked user messages

### ISSUE THREAD

**Issue by Deacllock** [42rave/transcendence#116]
Issue : 
1. You block a user from a channel
2. The user sends a message
3. You unblock the user
The message the user has sent  won't appear, however when you refresh the channel messages are updated and received.

Solution : Call backend to refresh the current channel when a user is unblocked. (only for the user unblocking someone)

**Comment by AdrianWeaver** [comment]
Let's say it's out of scope.
